midealocal/devices/fb/__init__.py

```diff
@@ -112,6 +112,14 @@ def set_attribute(self, attr: str, value: str | int | bool) -> None:
             setattr(message, str(attr), value)
         self.build_send(message)
 
+    def set_target_temperature(
```

Comment:
Why add `set_target_temperature` to FB device?

Reply:
BEacuse is needed on integration side as for this device we use directly set_attribute which has the wrong params

Reply:
#179